packages/api2/src/database/migrations/06_event_log_types.ts

```diff
@@ -40,6 +41,11 @@ const eventLogTypes = [
     label: 'Quantification',
     description: 'An action to quantify praise',
   },
+  {
+    key: EventLogTypeKey.PERIOD_SETTING,
+    label: 'Period Setting',
+    description: 'An action to period setting',
+  },
 ];
 
 const up = async (): Promise<void> => {
```

Comment:
You cannot change migrations "retroactively". If they have already run they cannot be changed. I removed change, let's use SETTING type for period setting changes as well.